mobile-app/app/screens/AppNavigator/screens/Dex/components/ViewPoolDetails.tsx

```diff
@@ -0,0 +1,335 @@
+        lhs={{
+          value: translate('screens/RemoveLiquidity', 'Tokens in {{token}}', {
+            token: pairData.tokenB.displaySymbol
+          }),
+          testID: `Pooled_${pairData.tokenB.displaySymbol}`,
+          themedProps: {
+            light: tailwind('text-mono-light-v2-500'),
+            dark: tailwind('text-mono-dark-v2-500')
+          }
+        }}
+        rhs={{
+          value: mappedPair?.tokenB.reserve ?? 0,
+          testID: `Pooled_${pairData.tokenB.displaySymbol}_${denominationCurrency}`,
+          usdTextStyle: tailwind('text-sm'),
+          usdAmount: getUSDValue(
+            new BigNumber(pairData.tokenB.reserve),
+            pairData.tokenB.symbol
+          )
+        }}
+        testID={`${pairInfo.displaySymbol}_pool_share_amount`}
+      />
+
+      {pairData?.apr?.total !== undefined && pairData?.apr?.total !== null && (
+        <ViewPoolAmountRow
+          label='APR'
+          amount={new BigNumber(isNaN(pairData.apr.total) ? 0 : pairData.apr.total).times(100).toFixed(2)}
+          valueThemeProps={{
+            dark: tailwind('text-green-v2'),
+            light: tailwind('text-green-v2')
+          }}
+          valueTextStyle={tailwind('font-semibold-v2')}
+          suffix='%'
+          testID={`${pairInfo.displaySymbol}_Apr`}
+        />
+      )}
+    </ThemedViewV2>
+  )
+}
+
+interface RemoveLiquidityDetailsProps {
+  pairData: PoolPairData
+  pairInfo: WalletToken
+}
+
+function RemoveLiquidityDetails ({ pairInfo, pairData }: RemoveLiquidityDetailsProps): JSX.Element {
+  const { poolpairs: pairs } = useSelector((state: RootState) => state.wallet)
+  const poolPairData = pairs.find(
+    (pr) => pr.data.symbol === (pairInfo as AddressToken).symbol
+  )
+  const mappedPair = poolPairData?.data
+  const toRemove = new BigNumber(1)
+    .times((pairInfo).amount)
+    .decimalPlaces(8, BigNumber.ROUND_DOWN)
+  const ratioToTotal = toRemove.div(mappedPair?.totalLiquidity?.token ?? 1)
+  const tokenATotal = ratioToTotal
+    .times(mappedPair?.tokenA.reserve ?? 0)
+    .decimalPlaces(8, BigNumber.ROUND_DOWN)
+  const tokenBTotal = ratioToTotal
+    .times(mappedPair?.tokenB.reserve ?? 0)
+    .decimalPlaces(8, BigNumber.ROUND_DOWN)
+  const lpPercentage = Number(pairInfo.amount) / Number(pairData.totalLiquidity.token)
+
+  const { getTokenPrice } = useTokenPrice()
+  const getUSDValue = (
+    amount: BigNumber,
+    symbol: string,
+    isLPs: boolean = false
+  ): BigNumber => {
+    return getTokenPrice(symbol, amount, isLPs)
+  }
+
+  return (
+    <ThemedViewV2 style={tailwind('mt-5')}>
+      <View style={tailwind('mb-5')}>
+        <NumberRowV2
+          lhs={{
+            value: translate('screens/RemoveLiquidity', 'Your LP Tokens'),
+            themedProps: {
+              light: tailwind('text-mono-light-v2-500'),
+              dark: tailwind('text-mono-dark-v2-500')
+            },
+            testID: 'lp_tokens_title'
+          }}
+          rhs={{
```

Comment:
![image](https://user-images.githubusercontent.com/13292662/183832765-86da03aa-bbaf-40c4-bbb5-a05da849c482.png)
why is it zero?